<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockGlyphSelector</class>
 <widget class="QDockWidget" name="DockGlyphSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font/Character selector</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBoxFont">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Font</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QFontComboBox" name="fontComboBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxFilters">
       <property name="title">
        <string>Filters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonResetFilters">
          <property name="text">
           <string>Reset Filters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="pageCategoriesFilter">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>285</width>
             <height>232</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Categories</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListView" name="listCategories">
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageScriptsFilter">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>285</width>
             <height>232</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Scripts</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QListView" name="listScripts">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>285</width>
             <height>232</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Decompositions</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QListView" name="listDecompositions">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageDecompositionFilter">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>285</width>
             <height>232</height>
            </rect>
           </property>
           <attribute name="label">
            <string>MSB/From/To</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QGroupBox" name="groupBoxMSB">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>MSB</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QCheckBox" name="checkBoxMsbHex">
                 <property name="text">
                  <string>Hexadecimal</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxMSB">
                 <property name="minimum">
                  <number>-1</number>
                 </property>
                 <property name="maximum">
                  <number>255</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxFromTo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>From/To</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBoxFromToHex">
                 <property name="text">
                  <string>Hexadecimal</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>69</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0">
                <widget class="QSpinBox" name="spinBoxFrom">
                 <property name="minimum">
                  <number>-1</number>
                 </property>
                 <property name="maximum">
                  <number>999999999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="2">
                <widget class="QSpinBox" name="spinBoxTo">
                 <property name="minimum">
                  <number>-1</number>
                 </property>
                 <property name="maximum">
                  <number>999999999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>69</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageCharactersFilter">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>285</width>
             <height>232</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Characters</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTextEdit" name="textFilter">
              <property name="frameShape">
               <enum>QFrame::Shape::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxCharacterTable">
       <property name="title">
        <string>Characters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tableCharacters"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonSelectAll">
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>172</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
